src/mutation.cpp

```diff
@@ -46,6 +47,17 @@ bool Character::has_base_trait(const std::string &b) const
     return my_traits.find( b ) != my_traits.end();
 }
 
+void update_vitamins( Character *ch ) {
```

Comment:
What's wit the pointer here? A reference should work just fine. (You can always convert the reference to a pointer using the the address-of operator inside the function.)